android/src/main/kotlin/com/qonversion/flutter/sdk/qonversion_flutter_sdk/QonversionPlugin.kt

```diff
@@ -183,8 +183,9 @@ class QonversionPlugin : MethodCallHandler, FlutterPlugin, ActivityAware {
         val productId = args["productId"] as? String ?: return result.noNecessaryDataError()
         val offerId = args["offerId"] as? String
         val applyOffer = args["applyOffer"] as? Boolean
+        val contextKeys = args["contextKeys"] as? List<String>
 
-        qonversionSandwich.purchase(productId, offerId, applyOffer, result.toJsonResultListener())
+        qonversionSandwich.purchase(productId, offerId, applyOffer, contextKeys, result.toJsonResultListener())
```

Comment:
in all the natives you change amount of expected parameters. Everything seem to be working with the old `purchase` method but to be sure - have you tested those calls?